robot_smach_states/src/robot_smach_states/world_model.py

```diff
@@ -223,7 +226,7 @@ class CheckEmpty(smach.State):
     """
     Check whether a volume of an entity is filled
     """
-    def __init__(self, robot, segmented_entity_ids_designator, entity_designator, volume, threshold=None):
+    def __init__(self, robot, segmented_entity_ids_designator, entity_designator, volume_designator, threshold=None):
```

Comment:
Since volume can also be a regular string I would keep the variable name as is. Just update the docstring